inspirehep/config.py

```diff
@@ -125,6 +125,114 @@ def _(x):
 USERPROFILES_EXTEND_SECURITY_FORMS = False
 USERPROFILES_SETTINGS_TEMPLATE = 'inspirehep_theme/accounts/settings/profile.html'
 
+# Collections
+# ===========
+COLLECTIONS_DELETED_RECORDS = '{dbquery} AND NOT collections.primary:"DELETED"'
```

Comment:
Currently, after @spirosdelviniotis work the  `DELETED` information corresponds to a flag being set to True in `deleted`.

I am not sure how to formulate this filter in Invenio query syntax though. (i.e. `deleted != True`)